rero_ils/es_templates/v7/record.json

```diff
@@ -1,5 +1,35 @@
 {
-  "index_patterns": "*",
+  "index_patterns": [
```

Comment:
What's happen for `operation_logs-*` mapping ? 
Does we need to duplicate this structure to allow default mapping ? I especially think about the `settings` section that isn't use for OpLog with your PR (If I understand correctly)

Reply:
The operation logs has his own template definition: https://github.com/rero/rero-ils/blob/staging/rero_ils/modules/operation_logs/es_templates/v7/operation_logs.json